app/Nova/Dashboards/JEDI.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Dashboards;
+
+use App\Nova\Metrics\AccessOverrides;
+use App\Nova\Metrics\LinkedGitHubAccounts;
+use App\Nova\Metrics\LinkedGoogleAccounts;
+use App\Nova\Metrics\PendingGitHubInvitations;
+use App\Nova\Metrics\SUMSUsers;
+use Illuminate\Http\Request;
+use Laravel\Nova\Dashboard;
+
+class JEDI extends Dashboard
```

Comment:
Can you set the URI key for this? Nova is adding dashes between all the letters.